src/app/win32_service.cc

```diff
@@ -93,30 +93,30 @@ void WINAPI I2PService::ServiceCtrlHandler(
 }
 
 I2PService::I2PService(
-    PSTR pszServiceName,
-    BOOL fCanStop,
-    BOOL fCanShutdown,
-    BOOL fCanPauseContinue) {
-  if (pszServiceName == NULL) {
+    PSTR psz_service_name,
+    BOOL can_stop,
+    BOOL can_shutdown,
+    BOOL can_pause_continue) {
+  if (psz_service_name == NULL) {
     m_Name = "";  // TODO(unassigned): why?
   } else {
-    m_Name = pszServiceName;
+    m_Name = psz_service_name;
   }
   m_StatusHandle = NULL;
   m_Status.dwServiceType = SERVICE_WIN32_OWN_PROCESS;
   m_Status.dwCurrentState = SERVICE_START_PENDING;
-  DWORD dwControlsAccepted = 0;
-  if (fCanStop)
-    dwControlsAccepted |= SERVICE_ACCEPT_STOP;
-  if (fCanShutdown)
-    dwControlsAccepted |= SERVICE_ACCEPT_SHUTDOWN;
-  if (fCanPauseContinue)
-    dwControlsAccepted |= SERVICE_ACCEPT_PAUSE_CONTINUE;
-  m_Status.dwControlsAccepted = dwControlsAccepted;
-  m_Status.dwWin32ExitCode = NO_ERROR;
-  m_Status.dwServiceSpecificExitCode = 0;
-  m_Status.dwCheckPoint = 0;
-  m_Status.dwWaitHint = 0;
+  DWORD dw_controls_accepted = 0;
+  if (can_stop)
```

Comment:
`fCanStop` is [hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation) for a "flag" type. Please feel free to remove all hungarian notation (like what you've done here).